### DISCUSSION TITLE
How to host on Vercel for a client?

### DISCUSSION THREAD

**Discussion by rossvold** [vercel/community#1295] [Help]
### Summary

If i have a client, that i want to host on Vercel, but the client already has his domain registered at their own domain registrar, what's the best solution here?

Can i point from their domainregistrar towards Vercel, or do i have to transfer the domain to Vercel?
Will this reduce loading speeds for the users of the website?

I have only made websites for myself and i have never thought of this.

**Comment by amyegan** [comment]
Thanks for reaching out @Rossvold! The client's domain can be directed to the Vercel project by setting DNS or Nameserver values. You can follow our "[add a custom domain](https://vercel.com/docs/concepts/projects/domains/add-a-domain#)" guide to configure domains and subdomains from a external domain registrar. This is a common setup and should have no impact on the loading speeds for users of the website.
